targets/bare.armv6m/freertos/sources

```diff
@@ -0,0 +1,14 @@
+VPATH := $(FREERTOSPATH)/Source: \
+         $(FREERTOSPATH)/Source/portable/GCC/ARM_CM3: \
```

Comment:
this is odd; why do we have a freertos directory in the bare.armv6m target?

Reply:
I agree that it is odd. We have it for the bare.armv7m target as well. That is where I copied it from.